src/screens/ProfileScreen/index.js

```diff
@@ -310,6 +312,12 @@ const ProfileScreen = ({route}) => {
       params: {user_id: userId, username}
     });
   };
+  const goToFollowers = (userId, username) => {
+    navigation.navigate('Followers', {
+      screen: 'TabFollowing',
+      params: {user_id: userId, username, isFollower: true}
+    });
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new function `goToFollowers` navigates to the 'Followers' screen with necessary parameters. However, it's not clear from this hunk whether the 'Followers' screen and 'TabFollowing' screen are properly set up in the navigation stack to accept these parameters and handle them correctly. Please ensure that these screens are configured correctly in your navigation setup.



<!-- This is an auto-generated comment by OSS CodeRabbit -->